### PR TITLE
Change pagerank updation rule to match GAP

### PR DESCRIPTION
**Pull request by gurbinder533**
GAP (http://gap.cs.berkeley.edu/) is considered standard for analytics.

### CODE REVIEW

File: lonestar/analytics/cpu/pagerank/PageRank-pull.cpp [hunk 1]
```diff
@@ -217,32 +218,31 @@ void computePRTopological(Graph& graph) {
...
-    if (delta <= tolerance || iteration >= maxIterations) {
+    if (accum.reduce() <= tolerance || iteration >= maxIterations) {
       break;
     }
-    max_delta.reset();
+    accum.reset();
 
   } ///< End while(true).
 
+  std::cout << "Rounds executed: " << iteration << "\n";
```
Comment (roshandathathri) [inline lonestar/analytics/cpu/pagerank/PageRank-pull.cpp:245]: We print this line in stats right? If we don't, then include it in stats. Please remove this line.

File: lonestar/analytics/cpu/pagerank/PageRank-pull.cpp [hunk 2]
```diff
@@ -63,7 +63,7 @@ void initNodeDataTopological(Graph& g) {
       galois::iterate(g),
       [&](const GNode& n) {
         auto& sdata = g.getData(n, galois::MethodFlag::UNPROTECTED);
-        sdata.value = INIT_RESIDUAL;
+        sdata.value = 1.0f / g.size();
```
Comment (roshandathathri) [inline lonestar/analytics/cpu/pagerank/PageRank-pull.cpp:66]: Declare a constant outside the do_all()

**Comment by l-hoang** [comment]
question: planning to do this for distributed as well?

**Comment by roshandathathri** [comment]
> question: planning to do this for distributed as well?

Our cpu, gpu, and distributed codes should be consistent with each other.

**Pull request merged**